frigate/object_processing.py

```diff
@@ -659,9 +659,14 @@ def on(self, event_type: str, callback: Callable[[dict], None]):
     def update(self, frame_time, current_detections, motion_boxes, regions):
         # get the new frame
         frame_id = f"{self.name}{frame_time}"
-        current_frame = self.frame_manager.get(
-            frame_id, self.camera_config.frame_shape_yuv
-        )
+
+        try:
+            current_frame = self.frame_manager.get(
+                frame_id, self.camera_config.frame_shape_yuv
+            )
+        except FileNotFoundError:
+            logger.error(f"Failed to get {frame_id} from SHM")
+            return
```

Comment:
Is return the behavior we would want here? Most of the code below doesn't depend on the frame.